files/KoreBuild/scripts/KoreBuild.psm1

```diff
@@ -308,10 +309,21 @@ function Set-KoreBuildSettings(
 
     if (!$ToolsSource) { $ToolsSource = 'https://aspnetcore.blob.core.windows.net/buildtools' }
 
+    if ($CI) {
+        $DotNetHome = Join-Path $RepoPath ".dotnet"
+
+        $env:CI = 'true'
+        $env:DOTNET_CLI_TELEMETRY_OPTOUT = 'true'
+        $env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE = 'true'
+        $env:NUGET_SHOW_STACK = 'true'
+        $env:NUGET_PACKAGES = Join-Path $RepoPath '.nuget'
```

Comment:
Nit: for consistency with Korebuild.sh, `Join-Paths $RepoPath ('.nuget', 'packages')`